metadata/system

```diff
@@ -0,0 +1 @@
+./store/4c1/4c13d11d-255a-4390-95a8-717f566ce360/
```

Comment:
not sure what's metadata but I guess it's .gitignore missing some files, maybe clickhouse